docs/deployment/deployment-checklist.md

```diff
@@ -167,6 +168,16 @@ The following checklist breaks down permissions required by Semgrep features.
 
 </TabItem>
 
+<TabItem value='az'>
+
+#### Azure DevOps
+
+| Feature  | Permission |
+| -------  | -------  |
+| Pull request (PR) comments.  | Able to create [user personal access tokens](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate). |
```

Comment:
Do we have any enterprise level access tokens? Or are the user personal access tokens applicable for orgs?

Reply:
Just PAT (Azure seems to have only two kinds: PAT and pipeline-related)